### DISCUSSION TITLE
questions about paper equation

### DISCUSSION THREAD

**Discussion by LuckyOne09** [nerfstudio-project/nerfacc#219] [Q&A] [answered]
Hi, thank you for your excellent work, but I have some questions regarding the derivation of an equation in the paper.
![image](https://github.com/KAIR-BAIR/nerfacc/assets/38714962/35088555-6cd5-42bf-b361-bf12c523aea2)
Thank you for your time and assistance. I appreciate your willingness to help me better understand your work.

**Comment by liruilong940607** [comment, marked as answer]
Hi yeah we omit some details in the paper, which we provide here:
<img width="1018" alt="Screen Shot 2023-05-13 at 9 32 41 AM" src="https://github.com/KAIR-BAIR/nerfacc/assets/10151885/24c444f5-d1be-4f4f-8e7b-5b4300228a03">

<img width="589" alt="Screen Shot 2023-05-13 at 9 29 40 AM" src="https://github.com/KAIR-BAIR/nerfacc/assets/10151885/84f0a3f9-4882-4c6b-84cd-120db6afde4d">

**Comment by LuckyOne09** [comment]
Wow, thank you for your so quick and detailed explanation. I can understand now and I am amazed that you could come up with such a clever deduction.
